Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
 WORKDIR /go/src/github.com/openshift/cincinnati-operator/
 COPY . .
-RUN go build -mod=vendor -o /tmp/build/update-service-operator github.com/openshift/cincinnati-operator
+RUN make GOBUILDFLAGS=-mod=vendor VERSION="$(git describe --abbrev=8 --dirty --always)" build
```

Comment:
`make GOBUILDFLAGS=-mod=vendor` does not look correct. This is a typo right?

Reply:
I guess it should be `GOFLAGS=${GOFLAGS:--mod=vendor}`

Reply:
It is dropping `-i`, which is part of [the default value][1].

[1]: https://github.com/openshift/cincinnati-operator/pull/88/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R12

Reply:
> I guess it should be `GOFLAGS=${GOFLAGS:--mod=vendor}`

Is anyone injecting a `GOFLAGS` environment variable into the Dockerfile's `RUN` context?  I don't see when that would happen, so generalizing to allow for it seems like overkill.